### PR TITLE
Mensa Balance Android Activity & Small changes

### PR DESCRIPTION
**Pull request by henry-herrmann**
This PR adds the native implementation for an Android activity to display the Mensa card balance even when the app is not running. 

Furthermore a series of smaller changes are introduced: 
- Rename the "Guide" to "Wallet" page 
- Update the tablet nav bar with the wallet page 
- Fix the alignment of the stacked cards on the wallet page 
- Add a source note at the end of each news article 
- Update the contents of the Campus ABC

### CODE REVIEW

**Review by domai-tb — CHANGES_REQUESTED** [review]
All adjustments on the Mensa and Guide / Wallet page works fine. But one huge problem occurs: The Feed stays empty / just show event items.

It is caused by the changes inside the NewsEntity class. The remote news feed uses the US date format, so changing the format inside the NewsEntity.fromXML-function will raise a FormatException. The format string inside the DateFormat(...) is the format to parse the publish date from, not parsing it to. So changing this string in any way will result in an exception. 

TL:DR: Reset the changes inside NewsEntit class; It causes FormatExceptions

File: lib/pages/feed/rubnews/news_entity.dart
Comment (domai-tb) [inline lib/pages/feed/rubnews/news_entity.dart:1]: These changes must be undone.

**Review by domai-tb — APPROVED** [review]
